guides/v2.2/cloud/project/project-multi-sites.md

```diff
@@ -8,157 +8,246 @@ functional_areas:
 
-{:.bs-callout .bs-callout-info}
+{:.bs-callout-info}
 For Pro, you must create a [Support ticket]({{ page.baseurl }}/cloud/trouble/trouble.html) to set up routes in the Staging or Production environment.
 
 #### To configure routes in an integration environment:
 
-1.  On your local workstation, log in as the [Magento file system owner]({{ page.baseurl }}/cloud/before/before-workspace-file-sys-owner.html).
-
-1.  Change to your {{site.data.var.ee}} base directory.
-
-1.  Open the `.magento/routes.yaml` file in a text editor.
-
-1.  Replace the contents with the following:
-
-    ```yaml
-    "http://{default}/":
-        type: upstream
-        upstream: "mymagento:php"
-
-    "https://{default}/":
-        type: upstream
-        upstream: "mymagento:php"
-
-    "http://*.{default}/":
-        type: upstream
-        upstream: "mymagento:php"
-
-    "https://*.{default}/":
-        type: upstream
-        upstream: "mymagento:php"
-    ```
-
-1.  Save your changes to the `routes.yaml` file.
+1. On your local workstation, log in as the [Magento file system owner]({{ page.baseurl }}/cloud/before/before-workspace-file-sys-owner.html).
+
+1. Change to your {{site.data.var.ee}} base directory.
+
+1. Open the `.magento/routes.yaml` file in a text editor.
+
+1. Replace the contents with the following:
+
+   ```yaml
+   "http://{default}/":
+       type: upstream
+       upstream: "mymagento:http"
+
+   "https://{default}/":
+       type: upstream
+       upstream: "mymagento:https"
+
+   "http://<second-site>.{default}/":
+       type: upstream
+       upstream: "mymagento:http"
+
+   "https://<second-site>.{default}/":
+       type: upstream
+       upstream: "mymagento:https"
+   ```
+
+1. Save your changes to the `routes.yaml` file.
+
+### Configure locations for shared domains {#locations}
+
+Where routes defines how the URLs are processed, the `web` property in the `.magento.app.yaml` file defines how your application is exposed to the web. Web _locations_ allows more granularity for incoming requests. For example, if your domain is `store.com`, you can use `/first` (default site) and `/second` for requests to two different stores that share that domain.
+
+```yaml
+web:
+    locations:
+        "/": &app
+            # The public directory of the app, relative to its root.
+            root: "pub"
+            passthru: "/index.php"
+            index:
+            - index.php
+            ...
+        "/media":
+            root: "pub/media"
+            ...
+        "/static":
+            root: "pub/static"
+            allow: true
+            scripts: false
+            passthru: "/front-static.php"
+            rules:
+                ^/static/version\d+/(?<resource>.*)$:
+                    passthru: "/static/$resource"
+        "/<second-site>": *app
+          ...
+```
+
+You can also add the directory to the `static` location rules. For example:
```

Comment:
if not to add directory for static, we need to change URLs in system configurations (Magento admin > Stores > Configurations > General > Web > (Secure) Base URL for Static View Files